ali.json

```diff
@@ -0,0 +1,28 @@
+{
+  "response": [
+    {
+        "maintainer": "BrunoRolak",
+        "oem": "Motorola",
+        "device": "Moto G6",
+        "filename": "crDroidAndroid-12.1-20220629-ali-v8.6.zip",
+        "download": "https://sourceforge.net/projects/crdroid/files/ali/8.x/crDroidAndroid-12.1-20220629-ali-v8.6.zip/download",
+        "timestamp": 1656542299,
+        "md5": "631393360c8bf29795228cf6c213d33a",
+        "sha256": "4df057385a81cf9cd8e3604b40d19ada77cc62ae05ff888229c3703305b5f316",
+        "size": 1017916329,
+        "version": "8.6",
+        "buildtype": "Monthly",
+        "forum": "https://forum.xda-developers.com/t/rom-12-1-crdroid-8-6-for-moto-g6-ali-official-64-bit.4463359/",
+        "gapps": "https://nikgapps.com/",
+        "firmware": "",
+        "modem": "",
+        "bootloader": "",
+        "recovery": "https://forum.xda-developers.com/t/twrp-unofficial-twrp-64-bit-3-6-1.4328145/",
+        "paypal": "",
+        "telegram": "https://t.me/Moto_G6",
+        "dt": "https://github.com/brunorolak/device_motorola_ali",
```

Comment:
use crDroid organization links for dt, common and kernel

Reply:
close these pr I'll make another witch right repo, my draught is jumping on me and I missed to wrote properlly

Reply:
oko fixed by forced push